### DISCUSSION TITLE
[Support]: last recognized person is not updated to the corresponding home assistant entity of the HA integration

### DISCUSSION THREAD

**Discussion by loadamasta** [blakeblackshear/frigate#12181] [Report a Bug]
### Describe the problem you are having

The picture of the last recognized person is not updated to the corresponding home assistant entity of the HA integration. In home assistant, the entity shows an old picture forever.

Restart of Frigate /addon in HA) as well as the Frigate integration in HA: no refresh of the image. It always remains the same.

### Steps to reproduce

none

### Version

2024.5.3

### Frigate config file

```yaml
not relevant
```

### Relevant log output

```shell
not relevant
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

reolink dorbell

**Comment by NickM-27** [comment]
I'm on the home assistant beta 2024.7 and not able to reproduce this issue
